helpers/pretests_check.rb

```diff
@@ -57,7 +57,7 @@ def self.request_to(path)
 
   def self.palladium_token?
     status = false
-    status = true if File.file?("#{ENV['HOME']}/.palladium/token")
+    status = true if File.size("#{ENV['HOME']}/.palladium/token") > 1
```

Comment:
File.size will raise an exception if there is no such file, so should be checked both file existing and file size 
See for example:
![Screenshot from 2020-12-03 10-33-59](https://user-images.githubusercontent.com/668524/100978196-1a049e00-3553-11eb-8f3c-07c3a01268d1.png)

Or better try to read this file and check content is not empty, and if read raise execption - return false

something like
```
def method
   return true unless File.read(name).empty?
rescue Errno::ENOENT
   false
end
```

Reply:
It is impossible to use the construction 
`return true unless File.read(name).empty?`
 because a non-empty file is created by 
`echo $PALLADIUM_TOKEN > ~/.palladium/token`

![screenshot](https://user-images.githubusercontent.com/60688343/101004150-89898600-3571-11eb-9c34-fe3a672a8761.png)